client/src/store/actions/authActions.tsx

```diff
@@ -10,6 +10,14 @@ export interface LoginData {
   remember: boolean;
 }
 
+export interface RegisterData {
```

Comment:
We have this in authTypes.tsx too, so we should remove this and import one from authTypes, same for LoginData

Reply:
Removed